app/code/Magento/Customer/view/frontend/templates/js/section-config.phtml

```diff
@@ -3,20 +3,25 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
 
-/** @var \Magento\Customer\Block\SectionConfig $block */
+use Magento\Customer\Block\SectionConfig;
+use Magento\Framework\Escaper;
+
+/** @var Escaper $escaper */
+/** @var SectionConfig $block */
 ?>
 <script type="text/x-magento-init">
     {
         "*": {
             "Magento_Customer/js/section-config": {
-                "sections": <?= /* @noEscape */ $this->helper(\Magento\Framework\Json\Helper\Data::class)->jsonEncode($block->getSections()) ?>,
-                "clientSideSections": <?= /* @noEscape */ $this->helper(\Magento\Framework\Json\Helper\Data::class)->jsonEncode($block->getClientSideSections()) ?>,
-                "baseUrls": <?= /* @noEscape */ $this->helper(\Magento\Framework\Json\Helper\Data::class)->jsonEncode(array_unique([
+                "sections": <?= /* @noEscape */ $escaper->escape->helper(\Magento\Framework\Json\Helper\Data::class)->jsonEncode($block->getSections()) ?>,
```

Comment:
This seems to be a mistake. @pykettk. All of these looks to have been added by mistake.